ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -95,10 +110,22 @@
          "to UMM-C had the following issues: "
          (string/join "; " warnings))))
 
+(defn- process-bulk-update-complete
+  "Check if the overall bulk update operation is complete and if do, re-index
+  provider collections"
+  [context provider-id task-id]
+  (let [task-status (data-bulk-update/get-bulk-update-task-status-for-provider context task-id)]
+    (when (= "COMPLETE" (:status task-status))
```

Comment:
Consider adding a constant for the complete status rather than using a magic string in the functions.

```
(def complete-status
    "Indicates bulk update operation finished successfully."
    "COMPLETE")
```